test/views/MoneyViewSpec.scala

```diff
@@ -50,17 +48,19 @@ class MoneyViewSpec extends ViewSpec with Mappings {
     val invalidSingleMoneyQuestion = formPageViewModelGen(singleQuestionGen(invalidMoneyForm))
     val viewModelGen = Gen.oneOf(singleMoneyQuestion, tripleMoneyQuestion, invalidSingleMoneyQuestion)
 
+    implicit val request = FakeRequest()
+
     "MoneyView" - {
 
-      act.like(renderTitle(viewModelGen)(view(_), _.title.key))
-      act.like(renderHeading(viewModelGen)(view(_), _.heading))
-      act.like(renderInputWithLabel(singleMoneyQuestion)("value", view(_), _.heading))
-      act.like(renderInputWithLabel(tripleMoneyQuestion)("value.1", view(_), _.page.fields.head.label))
-      act.like(renderInputWithLabel(tripleMoneyQuestion)("value.2", view(_), _.page.fields(1).label))
-      act.like(renderInputWithLabel(tripleMoneyQuestion)("value.3", view(_), _.page.fields(2).label))
-      act.like(renderErrors(invalidSingleMoneyQuestion)(view(_), _ => "money.error.required"))
-      act.like(renderForm(viewModelGen)(view(_), _.onSubmit))
-      act.like(renderButtonText(viewModelGen)(view(_), _.buttonText))
+      act.like(renderTitle(viewModelGen)(view(moneyForm, _), _.title.key))
+      act.like(renderHeading(viewModelGen)(view(moneyForm, _), _.heading))
+      act.like(renderInputWithLabel(singleMoneyQuestion)("value", view(moneyForm, _), _.heading))
+//      act.like(renderInputWithLabel(tripleMoneyQuestion)("value.1", view(_), _.page.fields.head.label))
```

Comment:
Same question for commented code as previously (checking commit by commit, may be resolved later)

Reply:
As above - yes, these tests will be replaced by `MultipleQuestionViewSpec`, on a separate branch (e.g. `PSR-405-unit-tests`)